tests/functional/cli/admin/test_service_list.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (C) 2019 OpenIO SAS, as part of OpenIO SDS
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3.0 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library.
+
+import tempfile
+from tests.utils import random_str
+from tests.functional.cli import CliTestCase
+
+
+class ServiceListTest(CliTestCase):
+
+    def test_meta2_list_containers(self):
+        opts = self.get_format_opts()
+        container = random_str(16)
+        output = self.openio('container create %s %s --oio-account %s'
+                             % (container, opts, self.account))
```

Comment:
Since we don't want to test `openio container {create,locate,delete}` and to avoid slowing the test, we should use `ObjectStorageApi.{container_create,container_get_properties,container_delete}`.